разбиение макета страницы 'services' на sections

NotACat1 opened the pull request:
0. header и footer оставлены не тронутыми, оставлено на разработчика, который создаст единый код для данных блоков.
1. `get-service`: в макете элемент `Frame 36040`
2. `list-services`: в макете элемент `Frame 98`

NotACat1 committed:
feat: разбиение макета страницы 'services' на sections

BJuice1984 commented on src/services.html:
```diff
@@ -0,0 +1,41 @@
+		<title></title>
+	</head>
+
+	<body class="page">
```
Отступы и табы слишком большие. в конфиге pritter указано 2 пробела. Точно ли настроен pritter?

NotACat1 replied:
Были выполнены настройки Pritter по файлу `HowToInstallPrettier.md`. Точно настройки `prettierrc.json` верные?

```json
// prettierrc.json
{
    "trailingComma": "es5",
    "tabWidth": 2,
    "semi": false,
    "singleQuote": true,
    "jsxSingleQuote": true,
    "arrowParens": "always",
    "useTabs": true
  }
```

![pritter настройки](https://github.com/NotACat1/RTE-Group/assets/113008873/88125a44-2204-469d-9f84-c95960890866)

BJuice1984 replied:
"tabWidth": 2 - говорит о настройке 2х пробелов. У тебя [настройка](https://glebbahmutov.com/blog/configure-prettier-in-vscode/) VSCode сделана под Pritter?

NotACat1 replied:
Правильно понимаю, что должно быть не два Tab, а два пробела?

NotACat1 replied:
Вывод терминала:

```
{
  "filepath": "d:\\Яндекс Практикум\\WEB\\спринт-6\\RTE-Group\\src\\services.html",
  "parser": "html",
  "trailingComma": "es5",
  "tabWidth": 2,
  "semi": false,
  "singleQuote": true,
  "jsxSingleQuote": true,
  "arrowParens": "always",
  "useTabs": true
}
```

BJuice1984 replied:
так, я, видимо, туплю. сейчас гляну настройки

BJuice1984 replied:
по табу должно быть два пробела. я ещё [тут](https://bobbyhadz.com/blog/vscode-change-indentation) посмотрел

BJuice1984 replied:
и ещё pritter должен автоматически исправлять код при сохранении - это в настройке vsCode

NotACat1 replied:
Есть вариант созвониться?

NotACat1 replied:
Найдена ошибка настройки `prettierrc.json`. Выставлена настройка на разрешение проставление Tab. 

```json
"useTabs": true // разрешение на проставление Tab
```

В vsCode даннной ошибки не видно из-за другой настройки:

```json
"tabWidth": 2 // ширина Tab = 2 пробелам
```

Данная настройка не влияет на GitHub, от чего здесь классическая ширина Tab = 4 пробелам.

BJuice1984 commented on src/services.html:
```diff
@@ -0,0 +1,41 @@
+		<div class="page__container">
+			<header></header>
+			<main>
+				<section class="" id="get-service"></section>
```
Предлагаю сразу задавать классы для секций и создавать в папке blocks, соответственно. Чтобы не получить конфликт имён

NotACat1 replied:
Предусматривать повторное использование блоков? Например слева текст, а справа форма (desktop) присутствует на страницах: `service`, `autopark`, `cases`

BJuice1984 replied:
да, предусмотри, пжл. В чатике напиши, пжл, сообщение и в задаче добавь, пжл, коммент об этом

BJuice1984 reviewed (commented):
1. Проверить настройку pritter
2. Создать классы секций

NotACat1 committed:
Merge branch 'dev' into services-html

NotACat1 committed:
feat: добавление класса `info-form`

Повторно использовать:
1. страница `service page`: `Frame 36063` (см. макет);
2. страница `autopark`: `Frame 36063` (см. макет);
3. страница `cases`: `Frame 36063` (см. макет);
4. страница `case page`: `Frame 36063` (см. макет);
5. страница `city page`: `Frame 36063` (см. макет);

NotACat1 committed:
feat: добавление класса `gallery-info-cards`

Повторно использовать:
1. страница `home`: `Frame 98` (см. макет);
2. страница `service page`: `Frame 36078` (см. макет);
3. страница `city page`: `Frame 98` (см. макет);
3. страница `about`: `Frame 36078` (см. макет);

NotACat1 committed:
perf: перемещено в папку `./src/page/`

NotACat1 committed:
feat: добавлены классы и заполнен `head`

BJuice1984 reviewed (approved):
Всё ОК
